### DISCUSSION TITLE
Add Cypress for testing example using mocking

### DISCUSSION THREAD

**Discussion by 0xle0ne** [0xle0ne/nextauri#19] [Ideas]
Hey guys if you want me to add `Cypress` to showcase how to test the Next.js project react to this discution !

**Comment by OskarHulter** [comment]
e2e testing would be nice to have, preferably under a flag to keep it flexible. 

Have you considered playwright? 

Not sure if there are extra requirements to handle desktop apps like tauri (new to me) but for my web use-cases PW has been a better fit.

Thanks for the package :)

**Comment by 0xle0ne** [comment]
Hey @OskarHulter
Thank you for sharing your thoughts!
I didn't know about playwright i will take a look!

You can use any e2e testing framework you want but it will require [mocking](https://tauri.app/v1/guides/testing/mocking/) tauri calls.
The only way to fully test is to use the [webdriver](https://tauri.app/v1/guides/testing/webdriver/introduction/) that Tauri offer.

**Reply by OskarHulter** [reply]
🙏 will report back if I find time to try.
